kymotracking: add bias reduction step into the centroid estimator (#452)
lumicks/pylake: changelog.md

#### Breaking changes

> * When performing particle tracking on kymographs, bias correction is now enabled by default. To disable bias correction, please specify `bias_correction=False` to `lk.track_greedy()` and `lk.refine_tracks_centroid()` if you wish to disable these. Without bias correction, kymographs with background will suffer from biased localization estimates.

Reviewer: ```suggestion
* When performing particle tracking on kymographs, bias correction is now enabled by default; without this correction, kymographs with high background signal will suffer from biased localization estimates. To disable bias correction, specify `bias_correction=False` to `lk.track_greedy()` and `lk.refine_tracks_centroid()`. ```

* Added API for notes, i.e. `file.notes` returns a dictionary of notes.
* Added option to add a scale bar to by providing a [`lk.ScaleBar()`](https://lumicks-pylake.readthedocs.io/en/latest/_api/lumicks.pylake.ScaleBar.html) to plotting or export functions.

> * Added extra method for centroid refinement that shrinks the window to reduce estimation bias.

Reviewer: this seems a bit oddly worded, at least for me I immediately thought of a method on a class. Maybe instead, we should mention explicitly the added `bias_correction` keyword arg?

Author: Good point. I changed it to:
```
* Implemented bias correction for centroid refinement that shrinks the window to reduce estimation bias. This bias correction can be toggled by passing a `bias_correction` argument to [`lk.track_greedy()`](https://lumicks-pylake.readthedocs.io/en/latest/_api/lumicks.pylake.track_greedy.html#) and [`lk.refine_tracks_centroid()`](https://lumicks-pylake.readthedocs.io/en/latest/_api/lumicks.pylake.refine_tracks_centroid.html).
```
